ammo-fe/cypress/integration/Error.cy.ts

```diff
@@ -3,22 +3,30 @@ import { bulletMock } from '../../src/shared/mocks/Bullet.mock';
 describe('Error boundary behaviors', () => {
     beforeEach(() => {
         cy.visit('/');
```

Comment:
For test speed, it might be faster to do :

before () => visit('/') and emitBullet()

The way the test are written, it will not have impact on them, and they'll run faster as we don't have to emit on every test (potentially an infinity based on how much we'll add in the future)